openweather-based-bidding/main.gs

```diff
@@ -35,86 +38,92 @@ function monitorWeatherAndSyncWithDV360() {
   // Get config from the spreadsheet
   const sheet = SpreadsheetApp.openById(configSpreadsheetId)
     .getSheetByName(configSpreadsheetName);
-  if (! sheet) {
+  if (!sheet) {
     throw 'Cannot find spreadsheet with the name: ' + configSpreadsheetName;
   }
 
-  // Filter out empty rows, since we took all rows from the specified range
-  const configFromSpreadsheet = sheet.getRange('A:I').getValues()
-    .filter((a) => a[0] || a[1]);
-
-  for (let i = 1; i < configFromSpreadsheet.length; i++) {
-    const lineItemId        = parseInt(configFromSpreadsheet[i][0]),
-        insertionOrderId    = parseInt(configFromSpreadsheet[i][1]),
-        advertiserId        = parseInt(configFromSpreadsheet[i][2]),
-        lat                 = configFromSpreadsheet[i][3],
-        lon                 = configFromSpreadsheet[i][4],
-        tempMin             = configFromSpreadsheet[i][5],
-        tempMax             = configFromSpreadsheet[i][6],
-        onlyIfRainingOrSnow = configFromSpreadsheet[i][7],
-        windMin             = configFromSpreadsheet[i][8];
+  for (let i = 1; i < rows.length; i++) {
+    const row = rows[i];
 
+    const lineItemId        = parseInt(row[config.get('col-line-item-id')]),
+        insertionOrderId    = parseInt(row[config.get('col-io-id')]),
+        advertiserId        = parseInt(row[config.get('col-advertiser-id')]),
+        lat                 = row[config.get('col-lat')],
+        lon                 = row[config.get('col-lon')],
+        tempMin             = row[config.get('col-temp-min')],
```

Comment:
How about rising en exception if a wrong input is given (the same for max temp., wind speed, lat., long.)? E.g.
```
tempMin             = parseFloat(row[config.get('col-temp-min')])
...
if (isNaN(tempMin)) {
  throw 'Please specify a numeric value for the "Min. Temperature"';
}
```